delius-core-dev/ansible/group_vars/all.yml

```
@@ -50,3 +50,5 @@ ndelius_oauth_config:
   username_attribute: username
   display_name: DOM1
   scope: delius
+  
+prepare_a_case_url: http://prepare-case-probation.service.justice.gov.uk/
```

```suggestion
prepare_a_case_url: https://prepare-case-probation.service.justice.gov.uk/
```

Should this be using `https` ?

Also, this looks like a production url - should we get confirmation that this is the correct url for the Delius Dev environment?

---

There was only a single URL given in the ticket which is the one in the config with no mention of what to do for the other environments (stage/test/prod etc). So I assumed it would just be the same url throughout all of them